react-native/READEME.md

```diff
@@ -0,0 +1,26 @@
+# React Native
+
+- Use React following the [React Guide](/react/)
+- Use [TypeScript](/typescript/)
+- Prefer using [core components and apis] over writing bespoke components.
+- Prefer grouping logic by feature rather than implementation detail.
+
+[core components and apis]: https://reactnative.dev/docs/components-and-apis
+
+## Style
+
+- Prefer [in-line and composable styles]
```

Comment:
I think the term "in-line" is confusing when used in conjunction with styles. It might be mistaken for in-line styles which is not what we are advocating. Maybe we can reword this ? Also it is worth mentioning that Stylesheets are [performant](https://stackoverflow.com/questions/39336266/react-native-inline-styles-and-performance)

Reply:
Agreed! What do you think about:
```
Prefer using StyleSheets
Prefer keeping styles colocated with components and composable
```

Reply:
Sounds good to me.